libs/blocks/icon-block/icon-block.css

```diff
@@ -103,14 +103,16 @@
   width: 60px;
 }
 
-.icon-block.inline.icon-height-32 .icon-area,
-.icon-block.inline.icon-height-32 .icon-area picture img {
-  height: 32px;
+.icon-block.inline.icon-height-40 .icon-area,
+.icon-block.inline.icon-height-40 .icon-area picture img {
+  min-width:40px;
+  height: 40px;
 }
```

Comment:
This change will require re-authoring of any page that uses the `icon-height-32` variant. Has this been communicated?

Reply:
I think the impact is low, but did this ever get communicated to the community?

Reply:
I posted in #milo-community relaying the breaking change.